opencti-platform/opencti-front/src/private/components/common/entreprise_edition/EnterpriseEdition.tsx

```diff
@@ -25,7 +25,7 @@ import type { Theme } from '../../../../components/Theme';
 const EnterpriseEdition = ({ message, feature }: { message?: string, feature?: string }) => {
   const theme = useTheme<Theme>();
 
-  const { isSensitiveModificationEnabled, isAllowed } = useSensitiveModifications();
+  const { isSensitive, isAllowed } = useSensitiveModifications('ce_ee_toggle');
```

Comment:
EntrepriseEditionButton at line 47 need to be updated to be disabled={!isAllowed & isSensitive}